workspaces/mend/plugins/mend-backend/src/service/auth.service.ts

```diff
@@ -0,0 +1,112 @@
+import jwt from 'jsonwebtoken';
+import { post } from '../api';
+import { caesarCipherDecrypt } from './auth.service.helpers';
+import {
+  JwtAuthToken,
+  JwtLicenceKeyPayload,
+  LoginSuccessResponseData,
+  MendConfig,
+  RefreshAccessTokenSuccessResponseData,
+} from './auth.services.types';
+
+enum AuthRoutes {
+  LOGIN = '/login',
+  REFRESH_TOKEN = '/login/accessToken',
+}
+
+export class MendAuthSevice {
+  private static authToken = '';
+  private static refreshToken = '';
+  private static baseUrl = '';
+  private static clientEmail = '';
+  private static clientKey = '';
+  private static clientUrl = '';
+  private static clientName = '';
+  private static clientUuid = '';
+
+  constructor(config: MendConfig) {
+    MendAuthSevice.baseUrl = config.baseUrl;
+    this.getConfig(config.activationKey);
```

Comment:
For config you should follow the patterns established here: https://backstage.io/docs/conf/defining.